ssh/ssh2ws.go

```diff
@@ -50,6 +50,22 @@ func HandlerWebsocket(ws *websocket.Conn) {
 		}
 
 		key, err := apiClient.GetPublicKey(fingerprint, device.TenantID)
+		// TODO This is just a draft
+		username := strings.Split(user, "@")[0] // username without '@'
+		var contains func(item string, items []string) bool
+		contains = func(item string, items []string) bool { // TODO Finding a function to check
```

Comment:
I guess that just a `for` could be more preformat, but I used recursion to avoid state variable.